### PR TITLE
Central config of doc link generation

### LINKED ISSUE #300: central config of doc link generation
Assuming we will link more and more doc links we need a central link generation module to have one single place where we change and/or inject the version

bildschirmfoto von 2016-04-06 22-19-31

### PR DESCRIPTION
**Pull request by VicDeo**
Closes #300

### CODE REVIEW

File: src/Command/CheckSystemCommand.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ protected function configure(){
 	protected function execute(InputInterface $input, OutputInterface $output){
 		$locator = $this->container['utils.locator'];
 		$fsHelper = $this->container['utils.filesystemhelper'];
-		$occRunner = $this->container['utils.occrunner'];
```
Comment (PVince81) [inline src/Command/CheckSystemCommand.php]: is that gone / old ?

**Comment by PVince81** [comment]
Code looks good otherwise :+1: 

@DeepDiver1975 second review ?

File: src/Command/CheckSystemCommand.php (hunk 1 above)
Comment (VicDeo) [inline src/Command/CheckSystemCommand.php]: @PVince81 yep, looks like this

Comment (VicDeo) [inline src/Command/CheckSystemCommand.php]: @PVince81 `$occRunner` is not used below in this method, so I just removed it

**Comment by PVince81** [comment]
Second review ? @DeepDiver1975 @LukasReschke

**Comment by VicDeo** [comment]
Rebased :speak_no_evil:

**Pull request merged**